src/core/geometry/qgsinternalgeometryengine.cpp

```diff
@@ -1503,6 +1503,13 @@ QgsGeometry QgsInternalGeometryEngine::orientedMinimumBoundingBox( double &area,
     pt1 = hull->vertexAt( vertexId );
   }
 
+  if ( width > height )
```

Comment:
Can you add a test for this case?

Reply:
Sure thing, I'll have one done as soon as I can!